scripts/release.py

```diff
@@ -22,6 +22,11 @@ def announce(version):
 
     contributors = set(stdout.splitlines())
 
+    # remove strings within contributors that have substring "[bot]"
+    for name in contributors:
```

Comment:
```suggestion
    for name in list(contributors):
```

Otherwise we will change the size of the set during iteration and raise an error.

Reply:
But probably it is easier to update line 23 to:

```python
contributors = set(x for x in stdout.splitlines() if "[bot]" not in x)
```

Reply:
Yeah I agree. I think the one-liner works well and looks nicer. Should I make a new commit?

Reply:
We typically squash PRs into a single commit when merging, so feel free to just push a second commit to the same branch (which will automatically update the PR).

As an aside, I'd probably use `name` instead of `x`, and `if not name.endswith("[bot]")` as I think it's always used as a suffix.